clients/python/llmengine/file.py

```diff
@@ -0,0 +1,192 @@
+from llmengine.api_engine import DEFAULT_TIMEOUT, APIEngine
+from llmengine.data_types import (
+    DeleteFileResponse,
+    GetFileContentResponse,
+    GetFileResponse,
+    ListFilesResponse,
+    UploadFileResponse,
+)
+
+
+class File(APIEngine):
+    """
+    File API. This API is used to upload private files to Scale so that fine-tunes can access them for training and validation data.
```

Comment:
We shouldn't mention Scale here, because this API will eventually work for self-hosted users too. The client can after all work in dual modes, talking to either the Scale-hosted or self-hosted LLM Engine server.